sdks/unity/AgonesSdk.cs

```diff
@@ -0,0 +1,239 @@
+﻿// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace Agones
+{
+    /// <summary>
+    /// Agones SDK for Unity.
+    /// </summary>
+    public class AgonesSdk : MonoBehaviour
+    {
+        [Range(1, 5)]
+        public float healthIntervalSecond = 5.0f;
+        public bool healthEnabled = true;

```

Comment:
You could reduce the number of fields here if this was a property that checked if `healthIntervalSecond` is > 0 (and document accordingly).